src/app/shared/mocks/vendors.ts

```diff
@@ -235,13 +224,12 @@ export default [
     name: 'Rockplay',
     title: 'Центр музыкальных развлечений',
     description: `Центр музыкальных развлечений «ROCKPLAY» - это уникальная игровая комната в Гродно. Только у нас Вы сможете сыграть в популярные «музыкальные симуляторы рок-группы» Guitar Hero и Rock Band, используя редчайшие контроллеры в виде реплик гитар известных музыкантов, ударной установки, клавиш, микрофонов, профессиональной акустики и другого оборудования, которое позволит себя почувствовать настоящей рок-звездой. Здесь можно в полной мере насладиться реалистичной атмосферой концерта и сыграть в собственной рок-группе из своих друзей, причем для этого вам НЕ НУЖНО уметь играть или иметь музыкальное образование. Достаточно справиться лишь с 5-ю цветными кнопками, которые, к примеру, имитируют удар по барабану или зажатый аккорд на гитаре, а когда и какую кнопку нажать - вы увидите на экране.  Впрочем, лучше один раз попробовать, чем читать о том, что невозможно объяснить) Так же у нас можно спеть рок-караоке, поиграть в танцевальные и спортивные игры с датчиком движения Kinect или просто весело провести время.`,
-    website: 'rockplay.by',
+    website: 'http://rockplay.by',
```

Comment:
Why do you use in website 'http'?In another mocks  you use 'https'.

Reply:
There I'm using real websites, I don't know why they still use http